modules/auth/proc-authenticationg-with-the-guest-user.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: PROCEDURE
+[id="authenticating-with-the-guest-user_{context}"]
+= Authenticating with the Guest user
+
+To start exploring {product} features, do not worry about setting up authentication: after the default installation, you can authenticate with the Guest user, and use all the application features.
```

Comment:
Similar to PR #460, I'd suggest something like this...
```suggestion
To explore {product-short} features, you do not need to configure authentication and authorization. By default, {product-short} allows you to authenticate with the Guest user which enables you to access authorized features associated with the guest user role.
```

Reply:
```suggestion
To explore {product-short} features, you can skip configuring authentication and authorization. 
After the default installation, you can log in as a Guest user and access all the features.
```